core-services/release-controller/_releases/priv/release-ocp-4.14-multi.json

```diff
@@ -55,6 +55,14 @@
                 "name": "periodic-ci-openshift-multiarch-master-nightly-4.14-ocp-e2e-aws-ovn-heterogeneous-priv"
             }
         },
+        "multi-gcp-ovn-upgrade": {
```

Comment:
@danilo-gemoli do we know in general why these priv jobs are all disabled?

Reply:
https://github.com/openshift/release/blob/d38f6f6b4c3082364e1c207f5fbb20fb1fde69e8/hack/generators/release-controllers/generate-release-controllers.py#L89-L90